testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/management/deployments/EjbJarRuntimeResourceTestBase.java

```diff
@@ -72,8 +75,36 @@
     private static final String[] POOL_ATTRIBUTES =
             {"pool-available-count", "pool-create-count", "pool-current-size", POOL_NAME, "pool-max-size", "pool-remove-count"};
 
-    private static final String[] TIMER_ATTRIBUTES = { "time-remaining", "next-timeout", "calendar-timer"};
-    private static final String[] SCHEDULE_ATTRIBUTES = { "day-of-month", "day-of-week", "hour", "minute", "year", "timezone", "start", "end" };
+    private static final String[] TIMER_ATTRIBUTES = {"time-remaining", "next-timeout", "calendar-timer"};
+    private static final String[] SCHEDULE_ATTRIBUTES = {"day-of-month", "day-of-week", "hour", "minute", "year", "timezone", "start", "end"};
+
+    private static final String JNDI_NAMES = "jndi-names";
+    private static final String BUSINESS_LOCAL = "business-local";
+    private static final String BUSINESS_REMOTE = "business-remote";
+    private static final String TIMEOUT_METHOD = "timeout-method";
+    private static final String ASYNC_METHODS = "async-methods";
+    private static final String TRANSACTION_TYPE = "transaction-type";
+
+    // MDB specific resource names
+    private static final String ACTIVATION_CONFIG = "activation-config";
+    private static final String MESSAGING_TYPE = "messaging-type";
+    private static final String MESSAGE_DESTINATION_TYPE = "message-destination-type";
+    private static final String MESSAGE_DESTINATION_LINK = "message-destination-link";
+
+    // STATEFUL specific resource names
+    private static final String STATEFUL_TIMEOUT = "stateful-timeout";
+    private static final String AFTER_BEGIN_METHOD = "after-begin-method";
+    private static final String BEFORE_COMPLETION_METHOD = "before-completion-method";
+    private static final String AFTER_COMPLETION_METHOD = "after-completion-method";
+    private static final String PASSIVATION_CAPABLE = "passivation-capable";
+    private static final String REMOVE_METHODS = "remove-methods";
+    private static final String RETAIN_IF_EXCEPTION = "retain-if-exception";
+    private static final String BEAN_METHOD = "bean-method";
+
+    // SINGLETON specific resource names
+    private static final String DEPENDS_ON = "depends-on";
+    private static final String INIT_ON_STARTUP = "init-on-startup";
+    private static final String CONCURRENCY_MANAGEMENT_TYPE = "concurrency-management-type";
 
```

Comment:
@chengfang I can't find tests related to `Bean statistics (beans in use, access count, idle count, ...)`, can you point me at them?

Reply:
@tommaso-borgato  these bean stats are existing features (already there before we implement this feature), so they are not covered by this PR.